src/filesys.cpp

```diff
@@ -395,6 +395,19 @@ void GetRecursiveSubPaths(const std::string &path, std::vector<std::string> &dst
 	}
 }
 
+void GetRecursiveSubDirs(const std::string &path, std::vector<std::string> &dst)
```

Comment:
This function is to find directories, not files. Otherwise call the function `GetRecursiveSubDirTree` or similar